charts/flagsmith/templates/deployment-api.yaml

```diff
@@ -148,9 +148,12 @@ spec:
 {{- with .Values.api.extraContainers }}
 {{- toYaml . | nindent 6 }}
 {{- end }}
-{{- if .Values.api.influxdbSetup.enabled }}
       volumes:
```

Comment:
Does this pass validation if `volumes` is empty which seems likely now with the new logic. I imagine it will, just double checking.

Reply:
It does as proved by the workflow test job.